component-starter-server/src/main/java/org/talend/sdk/component/starter/server/service/info/ServerInfo.java

```diff
@@ -159,7 +159,7 @@ private void refresh() {
             parser.parse(metadata, handler);
             if (handler.release != null) {
                 final String release = handler.release.toString();
-                if (!release.equals(snapshot.getKit())) {
+                if (!release.equals(snapshot.getKit()) && !"10.57.0".equals(release)) {
```

Comment:
What's the magic constant?

Reply:
This is not a magic constant, we want to exclude this release. It was released by Axel for testing stuffs and we cannot delete it and unfortunately it is considered as latest in maven: see https://repo.maven.apache.org/maven2/org/talend/sdk/component/component-starter-server/maven-metadata.xml